prisma/migrations/20240423092200_add_rateprovider_data/migration.sql

```diff
@@ -0,0 +1,15 @@
+-- CreateTable
+CREATE TABLE "PrismaPriceRateProviderData" (
+    "chain" "Chain" NOT NULL,
+    "rateProviderAddress" TEXT NOT NULL,
+    "tokenAddress" TEXT NOT NULL,
+    "reviewed" BOOLEAN NOT NULL,
+    "name" TEXT,
+    "summary" TEXT,
+    "reviewUrl" TEXT,
+
+    CONSTRAINT "PrismaPriceRateProviderData_pkey" PRIMARY KEY ("chain","rateProviderAddress")
+);
+
+-- AddForeignKey
+ALTER TABLE "PrismaPriceRateProviderData" ADD CONSTRAINT "PrismaPriceRateProviderData_tokenAddress_chain_fkey" FOREIGN KEY ("tokenAddress", "chain") REFERENCES "PrismaToken"("address", "chain") ON DELETE RESTRICT ON UPDATE CASCADE;
```

Comment:
Prisma always adds a constrain with DELETE RESTRICT, but in most cases we want delete cascade, because data is replicated anyways, and cascade makes pruning easier.